### DISCUSSION TITLE
Next Major Release

### DISCUSSION THREAD

**Discussion by coolsam726** [savannabits/jig#12] [Ideas]
I have been thinking about the next major release of JIG and the more I think about it, the more I am convinced that it is time to re-write it from scratch and possibly change the stack. Reasons:
1. I think currently it relys too much on npm dependencies
2. Not everyone is comfortable with the Jetstream + Inertia.js stack
3. The package cannot be installed by drag-dropping without interfering with the system. This renders it almost impossible to install into an existing project.
4. Vue is huge compared to Alpine.js
Based on this, I am thinking of making it a purely blade-driven system with blade components and alpine.js.

What do you guys think about this change?

**Comment by mquevedob** [comment]
Hi

I am happy with vue + jetstream approach

**Reply by mquevedob** [reply]
Maybe make another project with blade and alpinejs
Vue is good for single page app

**Reply by coolsam726** [reply]
> 

I think I like this.

**Reply by coolsam726** [reply]
> Hi
> 
> I am happy with vue + jetstream approach

How about replacing Jetstream with Breeze?

**Reply by mquevedob** [reply]
sure, that should work too

**Reply by mquevedob** [reply]
another point in favor of Vue is that (at least from what I heard) is that it is possible to convert it to android and ios apps. I havent test it yet but if it really works it allows to grab your VUE project and directly turn it into android and ios native apps

**Reply by coolsam726** [reply]
> another point in favor of Vue is that (at least from what I heard) is that it is possible to convert it to android and ios apps. I havent test it yet but if it really works it allows to grab your VUE project and directly turn it into android and ios native apps

Sure this works with Nativescript, but our code is a hybrid (by the fact that we are using Inertia) and I am not sure it can be easily convertible, but why not? It is something worth giving a try.
